projects/hipblaslt/tensilelite/Tensile/Components/CustomSchedule.py

```diff
@@ -537,6 +537,54 @@ def _get_schedule_256x256x64_16bit(kernel, useLDSTr, TLDS):
     opt1 = ScheduleInfo(2, numMfma, optSchedule, syncCode, nglshift, nllshift)
     return True, opt1
 
+def _get_schedule_256x160x64_16bit(kernel, useLDSTr, TLDS):
+    kernel["MfmaInitCVgprs"] = True
+    nglshift = nllshift = 0 # vmcnt shift for ngl and nll
+    if isNN(kernel) and useLDSTr and TLDS==1:
+        kernel["SwapGlobalReadOrder"] = True
+        optSchedule = {
+            'SYNC'   : [[-1,
+            12,12, # Wait for LRB0
+            24, 24,# Wait LRA0.
+            41,41,
+            61, 61 # Wait previous GR.
+            ]],
+            # Addr. update (be done before GRA/GRB).
+            'GRIncA' : [[0,1,2,3,4,5,6,7,8]],
+            'GRIncB' : [[9,10,10,10,13,14,15,16,17]],
+            # Current iteration.
+            'LRA0'   : [[5,5,7,7,9,9,11,11,13,13,15,15,17,18,19,20]],
+            'LRB0'   : [[0,0,1,2,3]],
+            # Buffer loads.
+            'GRB'    : [[30,30, 33,33, 36,36, 52,52, 56,56, 60,61, 76,77, 78,78]],
+            'GRA'    : [[11,12, 16,16, 20,20, 23,25, 26, 28]], 
+            # Prefetch next iteration.
+            'LRA1'   : [[62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77]],
+            'LRB1'   : [[41,42,43,44,45]],
+            'LRSA'   : [[39]],
+            'LRSB'   : [[39]],
+            'LWSA'   : [[60]],
+            'LWSB'   : [[60]],
+            'LCC'   : [[79, 79]], # Loop control
+        }
+        syncCode = [SWaitCnt(dscnt=0, vlcnt=-1, vscnt=-1, comment="Wait for LRA1 LRB1"),
+                    SWaitCnt(dscnt=8, vlcnt=-1, vscnt=-1, comment="Wait for LRB0"),
+                    SBarrier(comment=""),
+                    SWaitCnt(dscnt=0, vlcnt=-1, vscnt=-1, comment="Wait for previous GRB to complete"),
+                    SBarrier(comment=""),
+                    SWaitCnt(dscnt=-1, vlcnt=(13+8-5), vscnt=-1, comment="Wait for previous GRB to complete"),
+                    SBarrier(comment=""),
+                    SWaitCnt(dscnt=-1, vlcnt=(13+10-13), vscnt=-1, comment="Wait for previous GRA to complete"),
+                    SBarrier(comment="")]
+        nglshift = nllshift = 13
+    else:
+        return False, None
+
+
+    numMfma = 80
+    opt1 = ScheduleInfo(2, numMfma, optSchedule, syncCode, nglshift, nllshift)
```

Comment:
```suggestion
    opt1 = ScheduleInfo(1, numMfma, optSchedule, syncCode, nglshift, nllshift)
```

Or why is `numCodePaths` 2 here? My rough understanding was that >1 was for wave specialization, but all the fields (like LRA0) are lists of only a single list, which suggests all subgroups have the exact same instruction order?

(@sebvince)